app/models/user.rb

```diff
@@ -1,2 +1,12 @@
 class User < ActiveRecord::Base
+  def self.find_or_create_from_auth(auth)
```

Comment:
Yeah, i guess I don't see an issue with this. There's probably something you could do with generating the attributes to be saved in a separate method... but kind of seems overkill, and would probably just make things less clear. :-1: :dog: